packages/gp2-components/src/molecules/__tests__/ShareOutputButton.test.tsx

```diff
@@ -0,0 +1,48 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import ShareOutputButton from '../ShareOutputButton';
+
+describe('ShareOutputButton', () => {
+  it('should render a button', () => {
+    render(<ShareOutputButton entityType={'workingGroup'} id={'123'} />);
+    expect(
+      screen.getByRole('button', { name: /share an output/i }),
+    ).toBeVisible();
+  });
+  it('renders the right links', () => {
+    render(<ShareOutputButton entityType={'workingGroup'} id={'123'} />);
+    userEvent.click(screen.getByRole('button', { name: /share an output/i }));
+    expect(
+      screen
+        .getAllByRole('link')
+        .map((element) => (element as HTMLAnchorElement).href),
+    ).toEqual([
+      expect.stringContaining('article'),
+      expect.stringContaining('code%2Fsoftware'),
```

Comment:
What does %2F means in this case? From what I found out, it means a forward slash, in case it's in a query string, but want to make sure if it's the case